shell/cloud-credential/generic.vue

```diff
@@ -23,10 +24,15 @@ export default {
   async fetch() {
     let keyOptions = [];
 
-    const { normanSchema } = this;
+    const normanType = this.$store.getters['plugins/credentialFieldForDriver'](this.driverName);
+    const configId = `${ normanType }credentialconfig`;
+
+    this.normanSchema = await this.$store.dispatch('rancher/find', {
+      type: SCHEMA, id: configId, opt: { url: `/v3/schemas/${ configId }`, force: true }
```

Comment:
looking at this now, i've got my suggestion wrong. we don't need to `force` this, if the resource isn't in the cache the ui will go out and fetch it. we would only need to `force` if we already had the schema but it wasn't yet correctly populated (which i don't think is a thing, but worth checking?).

This would remove a blocking http request every time the user navs creates/edits that cloudcred type

Reply:
Ahhh good call! I checked and this does work without using `force`. I've updated the PR accordingly.